### PR TITLE
fix(ReportDetails): RHICOMPL-913 -Update policy details

### PR DESCRIPTION
**Pull request by bastilian**
This updates the policy details on the report details page to align more with the mockups:

Screenshot 2020-12-17 at 10 59 23

### CODE REVIEW

File: src/SmartComponents/ReportDetails/ReportDetails.js [hunk 1]
```diff
@@ -204,6 +204,7 @@ export const ReportDetails = ({ route }) => {
                         <SystemsTable
                             showOnlySystemsWithTestResults
                             compliantFilter
+                            remediationsEnabled={ false }
```
Comment (vkrizan) [inline src/SmartComponents/ReportDetails/ReportDetails.js:207]: Is this a deliberate change?

File: src/PresentationalComponents/ReportDetailsDescription/ReportDetailsDescription.js [hunk 2]
```diff
@@ -14,51 +14,72 @@ import {
...
+            <Dt>
+                <Text className='ins-c-non-bold-h2'>
+                    Policy details
+                </Text>
+            </Dt>
+            <Dt>
+                Operating system
+            </Dt>
+            <Dd>
+                RHEL { profile.majorOsVersion }
```
Comment (vkrizan) [inline src/PresentationalComponents/ReportDetailsDescription/ReportDetailsDescription.js:61]: This is the only content change to match the mocks. Is that true?

**Review by vkrizan — COMMENTED** [review]
The `SystemsTable` change seems like a mistake.

Would it be possible to highlight or isolate fixing changes from the refactoring?

Thank you.

File: src/SmartComponents/ReportDetails/ReportDetails.js (hunk 1 above)
Comment (bastilian) [reply src/SmartComponents/ReportDetails/ReportDetails.js]: Good catch. This was just accidentally committed. Removed it.

File: src/PresentationalComponents/ReportDetailsDescription/ReportDetailsDescription.js (hunk 2 above)
Comment (bastilian) [reply src/PresentationalComponents/ReportDetailsDescription/ReportDetailsDescription.js:61]: This is the only content change yes.

**Review by vkrizan — APPROVED** [review]

**Pull request merged**